praw/config.py

```diff
@@ -56,11 +64,13 @@ def short_url(self) -> str:
             raise ClientException("No short domain specified.")
         return self._short_url
 
-    def __init__(self, site_name: str, **settings: str):
+    def __init__(
+        self, site_name: str, interpolation: str = None, **settings: str
```

Comment:
Can we call this name `config_interpolation` so that it is clear that it applies to config?

Reply:
I wasn't sure if this applied to the Reddit or Config class so I changed both.